modules/serverpod_auth/serverpod_auth_shared_flutter/pubspec.yaml

```diff
@@ -20,7 +20,7 @@ dependencies:
   shared_preferences: ^2.1.0
   cached_network_image: ^3.2.3
   image: ^4.0.15
-  file_picker: '>=8.0.0 <10.0.0'
```

Comment:
There has been two breaking changes since 8.0.0 regarding `pickFiles` and `allowCompression`, but neither is used by us, so this seems okay.